TAF_EasyRest/TestFramework/Tools/DBConnectionWrapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Npgsql;
+
+namespace TestFramework.Tools
+{
+    internal class DBConnectionWrapper
+    {
+
+        private static readonly string pg_connectionString = "Host=localhost;Username=admin;Password=12345678;Database=easyrest";
+
+        static void ShowHeaders(NpgsqlDataReader reader)
+        {
+            var columnNames = Enumerable.Range(0, reader.FieldCount)
+                        .Select(reader.GetName)
+                        .ToArray();
+            foreach (var column in columnNames) { Console.Write($"{column}\t"); }
+            Console.WriteLine();
```

Comment:
why do you need this?

Reply:
This method ShowHeaders() is always followed by another method ShowRows() - and they represent together the answer from the database query. So without the WriteLine() it won't be correctly displayed: the first cell of the table body will be present on the same line as the header.